src/main/java/com/devathon/preguntonic/config/WebSocketConfig.java

```diff
@@ -37,9 +37,10 @@ public void registerStompEndpoints(StompEndpointRegistry registry) {
   @Override
   public void configureMessageBroker(MessageBrokerRegistry registry) {
     registry
-        .setApplicationDestinationPrefixes("/app") // [1]The prefix for the application destination.
+        .setApplicationDestinationPrefixes(
+            "/preguntonic") // [1]The prefix for the application destination.
         .enableSimpleBroker(
-            "/room"); // [2]Enable the simple broker (some examples of destinations).
+            "/rooms"); // [2]Enable the simple broker (some examples of destinations).
```

Comment:
we are doing the frontend integration with previous version, So I propose make this change in another PR, so as not to delay the integration further and do this cleaning when we have more functionalities running.

Reply:
But in yours version u are using `/rooms`, so i change to `/rooms`
![image](https://github.com/Devathon-2024-team5/Preguntonic-backend/assets/39089003/1e35c984-d2f7-4bf3-a26f-ed0dbd689993)